docs/developer-docs/latest/development/plugins-development.md

```diff
@@ -72,3 +72,7 @@ Strapi provides programmatic APIs for plugins to hook into some of Strapi's feat
 Plugins can register with the server and/or the admin panel, by looking for entry point files at the root of the package:
   - `strapi-server.js` for the Server (see [Server API](/developer-docs/latest/developer-resources/plugin-api-reference/server.md)),
   - `strapi-admin.js` for the admin panel (see [Admin Panel API](/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.md)).
+
+::: strapi Custom fields plugins
+Plugins can also be used to add [custom fields](/developer-docs/latest/development/custom-fields/reference.md) to Strapi.
+:::
```

Comment:
What's the ::: markup used for?

Reply:
It's how custom Markdown components are instantiated in VuePress, the framework we use to render our documentation ^^ It's used for tabs, callouts, and other custom components I created such as API calls 😊 
(See the full list in our [style guide](https://twitter.com/darylginn/status/1559212268532862977) 🤓 ) 

`::: strapi` is for a special "Strapi," purple-themed callout that we mostly use to bring readers' attention to specific product features and/or give more context about them.

Reply:
By the way if you'd like to see what the documentation will look like on the website, Vercel auto-deploys the content of each PR on preview URLs. The link can be found at the bottom of the PR 👇
https://user-images.githubusercontent.com/4233866/184880184-8f156fbe-a04c-4e6b-9877-92f636f0a61d.mov